Documentation/UWP.md

```diff
@@ -0,0 +1,43 @@
+# Game class constructor
+Due to some UWP implementation details, Monogame has to construct your `Game` derived class by itself, using a static initializer `MonoGame.Framework.XamlGame<T>.Create(...)`.
```

Comment:
@cra0zy @Jjagg - We should be proof reading docs like this.  `MonoGame` not `Monogame` it is sort of bad to have our own name wrong in our own docs. :)

Reply:
I can fix it tomorrow, need to earn that hacktoberfest t-shirt :P

Reply:
@cra0zy Sorry, I couldn't help it since this was my bad -> #6461. You still deserve a t-shirt though, awesome work these last few days!

Reply:
What is hacktoberfest ?

Reply:
Someone mentioned it on MG discord yesterday, like make 5 PRs in October on any open source project and you get a special t-shirt. Mostly just there promoting open source. https://hacktoberfest.digitalocean.com